changelog.d/8887.bugfix

```diff
@@ -0,0 +1 @@
+Add `X-Robots-Tag` header to stop web crawlers from indexing media.
```

Comment:
Oops, I marked the earlier discussion here as resolved too early. Could you change this to a `.feature` file please?